### DISCUSSION TITLE
Replacing GPG-Key with the same one with extended expiration

### DISCUSSION THREAD

**Discussion by atwebm** [community/community#36258] [Code Security]
To have a Safety-Net, when I`m loosing my GPG-Keys, I set an Expiration 2 years in the Feature and extend it Yearly.
Could you please add the Option to overwrite an existing GPG-Key with the same one with newer Expiration-Setting.
The Manual Step with deleting the old one and importing the new one breaks the Validation of older Commits this is not good as the Expiry-Date is also be used to verify a Commit.

**Comment by ossanna16** [comment]
Hey there @atwebm and thank you for asking a good question! I'm glad you found our community 🙂
